compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -41,6 +41,12 @@ struct CallSite<'tcx> {
 
 impl<'tcx> MirPass<'tcx> for Inline {
     fn is_enabled(&self, sess: &rustc_session::Session) -> bool {
+        // FIXME(#127234): Coverage instrumentation currently doesn't handle
+        // inlined MIR correctly when MC/DC is enabled.
```

Comment:
I would like to suggest another change to this comment, to make it more understandable to other maintainers who don't necessarily know what MC/DC is:

```rust
        // FIXME(#127234): Coverage instrumentation currently doesn't handle inlined
        // MIR correctly when Modified Condition/Decision Coverage is enabled.
```

(Sorry for the inconvenience! I edited my old suggestion shortly after writing it, but it looks like you copied the old version and didn't see the new version.)

Reply:
FIXED. Sorry for my carelessness :relieved: